### DISCUSSION TITLE
FREQUENCYGROUP_4G9

### DISCUSSION THREAD

**Discussion by Dvv-STM** [iNavFlight/inav#10421] [General]
Hello, need to build vtx_frequency_group = FREQUENCYGROUP_4G9 (band Boscam_A, Boscam_B, Boscam_E, Fatshark, Racebanb, Lowrace, Boscam_X)

**Comment by mmosca** [comment]
Where is this legal?

**Comment by Dvv-STM** [comment]
This is legal in Ukraine

нд, 20 жовт. 2024 р. о 14:58 Marcelo Bezerra ***@***.***>
пише:

**Comment by mmosca** [comment]
While we fully support Ukraine's right for self defense against the aggressors, we don't have plans to support military applications of INAV.

**Comment by Dvv-STM** [comment]
For military use, Inav is not suitable (machine vision cannot be attached),
4g9 is produced by Chinese manufacturers to improve the quality of analog
video in interference conditions


ср, 23 жовт. 2024 р. о 13:39 Marcelo Bezerra ***@***.***>
пише:
